<launch>

    <node pkg="multilayer" name="multilayer_node" type="multilayer_node" output="screen">

        <!-- <remap from="/odom" to="/t265/odom/sample"/>
        <remap from="/depth" to="/D455/depth/image_rect_raw"/> -->

        <!-- <remap from="/odom" to="/pose"/>
        <remap from="/depth" to="/camera/depth/image"/> -->

        <!-- <remap from="/odom" to="/mavros/local_position/odom"/>
        <remap from="/depth" to="/realsense_plugin/camera/depth/image_raw"/> -->

        <!-- <remap from="/odom" to="/tesse/odom"/>
        <remap from="/depth" to="/tesse/depth_cam/mono/image_raw"/> -->

        <remap from="/odom" to="/aft_mapped_to_init"/>
        <remap from="/depth" to="/depth_image"/>
        <remap from="/pointcloud" to="/livox/pointcloud2"/>

        <param name="frame_id" value="camera_init" type="string"/>
        <param name="child_frame_id" value="aft_mapped" type="string"/>

        <param name="/paramfile/path" value="$(find multilayer)/config/pointcloud_multilayer_map.yaml" type="string"/>

    </node>

    <node pkg="multilayer" name="global_gridmap_node" type="global_gridmap_node" output="screen">
        <!-- 添加frame_id参数 -->
        <param name="frame_id" value="camera_init" type="string"/>
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find multilayer)/launch/pointcloud.rviz" required="true" />

</launch>
